src/translations/en.json

```
@@ -1250,7 +1250,8 @@
               "open": "Open",
               "add_dashboard": "Add dashboard"
             },
-            "confirm_delete": "Are you sure you want to delete this dashboard?",
+            "confirm_delete_title": "Delete {dashboard_title}?",
+            "confirm_delete_text": "Your dashboard will be permanently deleted from Home Assistant.",
```

```suggestion
            "confirm_delete_text": "Your dashboard will be permanently deleted.",
```
This is in Home Assistant.